MdeModulePkg/Include/Protocol/MediaSanitize.h

```diff
@@ -0,0 +1,173 @@
+/** @file
+  This file defines the Media Sanitize Protocol.
+
+  Copyright (c) Microsoft Corporation.

```

Comment:
Could you add a `<BR>` at the end for doxygen, please?

Reply:
Updated in the latest push. Thanks for reviewing.